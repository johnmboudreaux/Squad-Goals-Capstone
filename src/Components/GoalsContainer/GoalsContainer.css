.goal-body {
  text-align: center;
  background-color: #3782AD;
  box-shadow: 2px 2px 4px black;
  border-radius: 5px;
  color: #fff;
  width: 100%;
  margin: 10px auto 10px auto;
  opacity: .95;
  height: auto; }

.goal-body-title {
  color: #d2d3d3;
  font-weight: lighter;
  font-size: 1.5em;
  margin: 10px 0 15px 0; }

.goal-body-info {
  color: #d2d3d3;
  display: flex;
  flex-direction: column;
  font-size: .8em; }

.goal-body-info p {
  margin: 10px 0 10px 0; }

.goal-row a {
  color: #d2d3d3;
  display: flex;
  text-decoration: none; }
