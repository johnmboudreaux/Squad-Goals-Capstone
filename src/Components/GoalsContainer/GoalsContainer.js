import React from 'react';
import PropTypes from 'prop-types';
import GoalCard from '../GoalCard/GoalCard';

const GoalsContainer = ({ goalData, leaveGroup, userId }) => {
  const goalInfo = goalData.map((goal) => {
    return (
      <GoalCard
        leaveGroup={leaveGroup}
        userId={userId}
        goal={goal}
        key={`goal-${goal.id}`}
      />
    );
  });

  return (
    <div className="goal-body">
      <h2 className="goal-body-title">Goals</h2>
      <table className="goal-body-info">
        <tbody>
          <tr className="goal-row">
            <th>Goal Title</th>
            <th>Goal Time </th>
            <th>Goal Points </th>
            <th />
          </tr>
          { goalInfo }
        </tbody>
      </table>
    </div>
  );
};

GoalsContainer.propTypes = {
  goalData: PropTypes.arrayOf(PropTypes.object),
  leaveGroup: PropTypes.func,
  userId: PropTypes.number,
};

export default GoalsContainer;
