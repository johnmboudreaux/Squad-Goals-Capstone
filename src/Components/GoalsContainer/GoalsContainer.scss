@import '../../variables.scss';

.goal-body {
  text-align: center;
  background-color: $main;
  box-shadow: 2px 2px 4px black;
  border-radius: 5px;
  color: #fff;
  width: 100%;
  margin: 10px auto 10px auto;
  opacity: .95;
  height: auto;
  border-radius: 25px;
}

.goal-body-title {
  color: $white;
  font-weight: lighter;
  font-size: 1.5em;
  margin: 10px 0 15px 0;
}

.goal-body-info {
  color: $white;
  display: flex;
  flex-direction: column;
  font-size: .8em
}

.goal-body-info p {
  margin: 10px 0 10px 0;
}

.goal-row a {
  color: $white;
  display: flex;
  text-decoration: none;
}
