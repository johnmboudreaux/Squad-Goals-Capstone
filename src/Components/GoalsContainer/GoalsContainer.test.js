import { shallow } from 'enzyme';
import React from 'react';
import ReactDOM from 'react-dom';
import GoalsContainer from './GoalsContainer';
import goalData from '../../../data/test/goals_data_test';

describe('GoalsContainer snapshot', () => {
  it('should always match the snapshot', () => {
    const wrapper = shallow(<GoalsContainer
      goalData={goalData}
    />);

    expect(wrapper).toMatchSnapshot();
  });


  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<GoalsContainer
      goalData={goalData}
    />, div);
  });
});
