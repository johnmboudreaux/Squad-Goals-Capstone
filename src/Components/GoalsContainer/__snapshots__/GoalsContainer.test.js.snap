// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoalsContainer snapshot should always match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <GoalsContainer
    goalData={
      Array [
        Object {
          "conversation_id": 1,
          "creator_id": 1,
          "description": "to rule them all",
          "goal_points": 2300,
          "goal_time": "2018-01-12 15:00:00",
          "id": 1,
          "title": "chess tourney",
        },
        Object {
          "conversation_id": 2,
          "creator_id": 2,
          "description": "to skewer them all",
          "goal_points": 23000,
          "goal_time": "2018-01-13 16:00:00",
          "id": 2,
          "title": "darts tourney",
        },
        Object {
          "conversation_id": 3,
          "creator_id": 2,
          "description": "to pool them all",
          "goal_points": 230000,
          "goal_time": "2018-01-14 17:00:00",
          "id": 3,
          "title": "pool tourney",
        },
        Object {
          "conversation_id": 4,
          "creator_id": 3,
          "description": "to drink them all",
          "goal_points": 2300000,
          "goal_time": "2018-01-14 18:00:00",
          "id": 4,
          "title": "drinking tourney",
        },
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="goal-body-title"
        >
          Goals
        </h2>,
        <table
          className="goal-body-info"
        >
          <tbody>
            <tr
              className="goal-row"
            >
              <th>
                Goal Title
              </th>
              <th>
                Goal Time 
              </th>
              <th>
                Goal Points 
              </th>
              <th />
            </tr>
            <GoalCard
              goal={
                Object {
                  "conversation_id": 1,
                  "creator_id": 1,
                  "description": "to rule them all",
                  "goal_points": 2300,
                  "goal_time": "2018-01-12 15:00:00",
                  "id": 1,
                  "title": "chess tourney",
                }
              }
              leaveGroup={undefined}
              userId={undefined}
            />
            <GoalCard
              goal={
                Object {
                  "conversation_id": 2,
                  "creator_id": 2,
                  "description": "to skewer them all",
                  "goal_points": 23000,
                  "goal_time": "2018-01-13 16:00:00",
                  "id": 2,
                  "title": "darts tourney",
                }
              }
              leaveGroup={undefined}
              userId={undefined}
            />
            <GoalCard
              goal={
                Object {
                  "conversation_id": 3,
                  "creator_id": 2,
                  "description": "to pool them all",
                  "goal_points": 230000,
                  "goal_time": "2018-01-14 17:00:00",
                  "id": 3,
                  "title": "pool tourney",
                }
              }
              leaveGroup={undefined}
              userId={undefined}
            />
            <GoalCard
              goal={
                Object {
                  "conversation_id": 4,
                  "creator_id": 3,
                  "description": "to drink them all",
                  "goal_points": 2300000,
                  "goal_time": "2018-01-14 18:00:00",
                  "id": 4,
                  "title": "drinking tourney",
                }
              }
              leaveGroup={undefined}
              userId={undefined}
            />
          </tbody>
        </table>,
      ],
      "className": "goal-body",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Goals",
          "className": "goal-body-title",
        },
        "ref": null,
        "rendered": "Goals",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <tbody>
            <tr
              className="goal-row"
            >
              <th>
                Goal Title
              </th>
              <th>
                Goal Time 
              </th>
              <th>
                Goal Points 
              </th>
              <th />
            </tr>
            <GoalCard
              goal={
                Object {
                  "conversation_id": 1,
                  "creator_id": 1,
                  "description": "to rule them all",
                  "goal_points": 2300,
                  "goal_time": "2018-01-12 15:00:00",
                  "id": 1,
                  "title": "chess tourney",
                }
              }
              leaveGroup={undefined}
              userId={undefined}
            />
            <GoalCard
              goal={
                Object {
                  "conversation_id": 2,
                  "creator_id": 2,
                  "description": "to skewer them all",
                  "goal_points": 23000,
                  "goal_time": "2018-01-13 16:00:00",
                  "id": 2,
                  "title": "darts tourney",
                }
              }
              leaveGroup={undefined}
              userId={undefined}
            />
            <GoalCard
              goal={
                Object {
                  "conversation_id": 3,
                  "creator_id": 2,
                  "description": "to pool them all",
                  "goal_points": 230000,
                  "goal_time": "2018-01-14 17:00:00",
                  "id": 3,
                  "title": "pool tourney",
                }
              }
              leaveGroup={undefined}
              userId={undefined}
            />
            <GoalCard
              goal={
                Object {
                  "conversation_id": 4,
                  "creator_id": 3,
                  "description": "to drink them all",
                  "goal_points": 2300000,
                  "goal_time": "2018-01-14 18:00:00",
                  "id": 4,
                  "title": "drinking tourney",
                }
              }
              leaveGroup={undefined}
              userId={undefined}
            />
          </tbody>,
          "className": "goal-body-info",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <tr
                className="goal-row"
              >
                <th>
                  Goal Title
                </th>
                <th>
                  Goal Time 
                </th>
                <th>
                  Goal Points 
                </th>
                <th />
              </tr>,
              Array [
                <GoalCard
                  goal={
                    Object {
                      "conversation_id": 1,
                      "creator_id": 1,
                      "description": "to rule them all",
                      "goal_points": 2300,
                      "goal_time": "2018-01-12 15:00:00",
                      "id": 1,
                      "title": "chess tourney",
                    }
                  }
                  leaveGroup={undefined}
                  userId={undefined}
                />,
                <GoalCard
                  goal={
                    Object {
                      "conversation_id": 2,
                      "creator_id": 2,
                      "description": "to skewer them all",
                      "goal_points": 23000,
                      "goal_time": "2018-01-13 16:00:00",
                      "id": 2,
                      "title": "darts tourney",
                    }
                  }
                  leaveGroup={undefined}
                  userId={undefined}
                />,
                <GoalCard
                  goal={
                    Object {
                      "conversation_id": 3,
                      "creator_id": 2,
                      "description": "to pool them all",
                      "goal_points": 230000,
                      "goal_time": "2018-01-14 17:00:00",
                      "id": 3,
                      "title": "pool tourney",
                    }
                  }
                  leaveGroup={undefined}
                  userId={undefined}
                />,
                <GoalCard
                  goal={
                    Object {
                      "conversation_id": 4,
                      "creator_id": 3,
                      "description": "to drink them all",
                      "goal_points": 2300000,
                      "goal_time": "2018-01-14 18:00:00",
                      "id": 4,
                      "title": "drinking tourney",
                    }
                  }
                  leaveGroup={undefined}
                  userId={undefined}
                />,
              ],
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <th>
                    Goal Title
                  </th>,
                  <th>
                    Goal Time 
                  </th>,
                  <th>
                    Goal Points 
                  </th>,
                  <th />,
                ],
                "className": "goal-row",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Goal Title",
                  },
                  "ref": null,
                  "rendered": "Goal Title",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Goal Time ",
                  },
                  "ref": null,
                  "rendered": "Goal Time ",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Goal Points ",
                  },
                  "ref": null,
                  "rendered": "Goal Points ",
                  "type": "th",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": "th",
                },
              ],
              "type": "tr",
            },
            Object {
              "instance": null,
              "key": "goal-1",
              "nodeType": "function",
              "props": Object {
                "goal": Object {
                  "conversation_id": 1,
                  "creator_id": 1,
                  "description": "to rule them all",
                  "goal_points": 2300,
                  "goal_time": "2018-01-12 15:00:00",
                  "id": 1,
                  "title": "chess tourney",
                },
                "leaveGroup": undefined,
                "userId": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "goal-2",
              "nodeType": "function",
              "props": Object {
                "goal": Object {
                  "conversation_id": 2,
                  "creator_id": 2,
                  "description": "to skewer them all",
                  "goal_points": 23000,
                  "goal_time": "2018-01-13 16:00:00",
                  "id": 2,
                  "title": "darts tourney",
                },
                "leaveGroup": undefined,
                "userId": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "goal-3",
              "nodeType": "function",
              "props": Object {
                "goal": Object {
                  "conversation_id": 3,
                  "creator_id": 2,
                  "description": "to pool them all",
                  "goal_points": 230000,
                  "goal_time": "2018-01-14 17:00:00",
                  "id": 3,
                  "title": "pool tourney",
                },
                "leaveGroup": undefined,
                "userId": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "goal-4",
              "nodeType": "function",
              "props": Object {
                "goal": Object {
                  "conversation_id": 4,
                  "creator_id": 3,
                  "description": "to drink them all",
                  "goal_points": 2300000,
                  "goal_time": "2018-01-14 18:00:00",
                  "id": 4,
                  "title": "drinking tourney",
                },
                "leaveGroup": undefined,
                "userId": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "tbody",
        },
        "type": "table",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="goal-body-title"
          >
            Goals
          </h2>,
          <table
            className="goal-body-info"
          >
            <tbody>
              <tr
                className="goal-row"
              >
                <th>
                  Goal Title
                </th>
                <th>
                  Goal Time 
                </th>
                <th>
                  Goal Points 
                </th>
                <th />
              </tr>
              <GoalCard
                goal={
                  Object {
                    "conversation_id": 1,
                    "creator_id": 1,
                    "description": "to rule them all",
                    "goal_points": 2300,
                    "goal_time": "2018-01-12 15:00:00",
                    "id": 1,
                    "title": "chess tourney",
                  }
                }
                leaveGroup={undefined}
                userId={undefined}
              />
              <GoalCard
                goal={
                  Object {
                    "conversation_id": 2,
                    "creator_id": 2,
                    "description": "to skewer them all",
                    "goal_points": 23000,
                    "goal_time": "2018-01-13 16:00:00",
                    "id": 2,
                    "title": "darts tourney",
                  }
                }
                leaveGroup={undefined}
                userId={undefined}
              />
              <GoalCard
                goal={
                  Object {
                    "conversation_id": 3,
                    "creator_id": 2,
                    "description": "to pool them all",
                    "goal_points": 230000,
                    "goal_time": "2018-01-14 17:00:00",
                    "id": 3,
                    "title": "pool tourney",
                  }
                }
                leaveGroup={undefined}
                userId={undefined}
              />
              <GoalCard
                goal={
                  Object {
                    "conversation_id": 4,
                    "creator_id": 3,
                    "description": "to drink them all",
                    "goal_points": 2300000,
                    "goal_time": "2018-01-14 18:00:00",
                    "id": 4,
                    "title": "drinking tourney",
                  }
                }
                leaveGroup={undefined}
                userId={undefined}
              />
            </tbody>
          </table>,
        ],
        "className": "goal-body",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Goals",
            "className": "goal-body-title",
          },
          "ref": null,
          "rendered": "Goals",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <tbody>
              <tr
                className="goal-row"
              >
                <th>
                  Goal Title
                </th>
                <th>
                  Goal Time 
                </th>
                <th>
                  Goal Points 
                </th>
                <th />
              </tr>
              <GoalCard
                goal={
                  Object {
                    "conversation_id": 1,
                    "creator_id": 1,
                    "description": "to rule them all",
                    "goal_points": 2300,
                    "goal_time": "2018-01-12 15:00:00",
                    "id": 1,
                    "title": "chess tourney",
                  }
                }
                leaveGroup={undefined}
                userId={undefined}
              />
              <GoalCard
                goal={
                  Object {
                    "conversation_id": 2,
                    "creator_id": 2,
                    "description": "to skewer them all",
                    "goal_points": 23000,
                    "goal_time": "2018-01-13 16:00:00",
                    "id": 2,
                    "title": "darts tourney",
                  }
                }
                leaveGroup={undefined}
                userId={undefined}
              />
              <GoalCard
                goal={
                  Object {
                    "conversation_id": 3,
                    "creator_id": 2,
                    "description": "to pool them all",
                    "goal_points": 230000,
                    "goal_time": "2018-01-14 17:00:00",
                    "id": 3,
                    "title": "pool tourney",
                  }
                }
                leaveGroup={undefined}
                userId={undefined}
              />
              <GoalCard
                goal={
                  Object {
                    "conversation_id": 4,
                    "creator_id": 3,
                    "description": "to drink them all",
                    "goal_points": 2300000,
                    "goal_time": "2018-01-14 18:00:00",
                    "id": 4,
                    "title": "drinking tourney",
                  }
                }
                leaveGroup={undefined}
                userId={undefined}
              />
            </tbody>,
            "className": "goal-body-info",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <tr
                  className="goal-row"
                >
                  <th>
                    Goal Title
                  </th>
                  <th>
                    Goal Time 
                  </th>
                  <th>
                    Goal Points 
                  </th>
                  <th />
                </tr>,
                Array [
                  <GoalCard
                    goal={
                      Object {
                        "conversation_id": 1,
                        "creator_id": 1,
                        "description": "to rule them all",
                        "goal_points": 2300,
                        "goal_time": "2018-01-12 15:00:00",
                        "id": 1,
                        "title": "chess tourney",
                      }
                    }
                    leaveGroup={undefined}
                    userId={undefined}
                  />,
                  <GoalCard
                    goal={
                      Object {
                        "conversation_id": 2,
                        "creator_id": 2,
                        "description": "to skewer them all",
                        "goal_points": 23000,
                        "goal_time": "2018-01-13 16:00:00",
                        "id": 2,
                        "title": "darts tourney",
                      }
                    }
                    leaveGroup={undefined}
                    userId={undefined}
                  />,
                  <GoalCard
                    goal={
                      Object {
                        "conversation_id": 3,
                        "creator_id": 2,
                        "description": "to pool them all",
                        "goal_points": 230000,
                        "goal_time": "2018-01-14 17:00:00",
                        "id": 3,
                        "title": "pool tourney",
                      }
                    }
                    leaveGroup={undefined}
                    userId={undefined}
                  />,
                  <GoalCard
                    goal={
                      Object {
                        "conversation_id": 4,
                        "creator_id": 3,
                        "description": "to drink them all",
                        "goal_points": 2300000,
                        "goal_time": "2018-01-14 18:00:00",
                        "id": 4,
                        "title": "drinking tourney",
                      }
                    }
                    leaveGroup={undefined}
                    userId={undefined}
                  />,
                ],
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <th>
                      Goal Title
                    </th>,
                    <th>
                      Goal Time 
                    </th>,
                    <th>
                      Goal Points 
                    </th>,
                    <th />,
                  ],
                  "className": "goal-row",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Goal Title",
                    },
                    "ref": null,
                    "rendered": "Goal Title",
                    "type": "th",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Goal Time ",
                    },
                    "ref": null,
                    "rendered": "Goal Time ",
                    "type": "th",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Goal Points ",
                    },
                    "ref": null,
                    "rendered": "Goal Points ",
                    "type": "th",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": "th",
                  },
                ],
                "type": "tr",
              },
              Object {
                "instance": null,
                "key": "goal-1",
                "nodeType": "function",
                "props": Object {
                  "goal": Object {
                    "conversation_id": 1,
                    "creator_id": 1,
                    "description": "to rule them all",
                    "goal_points": 2300,
                    "goal_time": "2018-01-12 15:00:00",
                    "id": 1,
                    "title": "chess tourney",
                  },
                  "leaveGroup": undefined,
                  "userId": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "goal-2",
                "nodeType": "function",
                "props": Object {
                  "goal": Object {
                    "conversation_id": 2,
                    "creator_id": 2,
                    "description": "to skewer them all",
                    "goal_points": 23000,
                    "goal_time": "2018-01-13 16:00:00",
                    "id": 2,
                    "title": "darts tourney",
                  },
                  "leaveGroup": undefined,
                  "userId": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "goal-3",
                "nodeType": "function",
                "props": Object {
                  "goal": Object {
                    "conversation_id": 3,
                    "creator_id": 2,
                    "description": "to pool them all",
                    "goal_points": 230000,
                    "goal_time": "2018-01-14 17:00:00",
                    "id": 3,
                    "title": "pool tourney",
                  },
                  "leaveGroup": undefined,
                  "userId": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "goal-4",
                "nodeType": "function",
                "props": Object {
                  "goal": Object {
                    "conversation_id": 4,
                    "creator_id": 3,
                    "description": "to drink them all",
                    "goal_points": 2300000,
                    "goal_time": "2018-01-14 18:00:00",
                    "id": 4,
                    "title": "drinking tourney",
                  },
                  "leaveGroup": undefined,
                  "userId": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "tbody",
          },
          "type": "table",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
